pyirf/interpolation/moment_morph_interpolator.py

```diff
@@ -29,35 +29,40 @@ def _estimate_mean_std(bin_edges, bin_contents):
         the input template is =/= 0
     """
     # Create an 2darray where the 1darray mids is repeated n_template times
-    mids = np.broadcast_to(bin_center(bin_edges), bin_contents.shape)
+    mids = np.broadcast_to(bin_center(bin_edges), binned_pdf.shape)
+
+    width = get_bin_width(bin_edges, normalization)
+
+    # integrate pdf to get probability in each bin
+    probability = binned_pdf * width
     # Weighted averages to compute mean and std
-    mean = np.average(mids, weights=bin_contents, axis=-1)
+    mean = np.average(mids, weights=probability, axis=-1)
     std = np.sqrt(
-        np.average((mids - mean[..., np.newaxis]) ** 2, weights=bin_contents, axis=-1)
+        np.average((mids - mean[..., np.newaxis]) ** 2, weights=probability, axis=-1)
     )
 
     # Set std to 0.5*width for all those templates that have only one bin =/= 0. In those
     # cases mids-mean = 0 and therefore std = 0. Uses the width of the one bin with
-    # bin_content!=0
+    # binned_pdf!=0
     mask = std == 0
     if np.any(mask):
         # Create array of bin_widths and clone for each template
         width = np.diff(bin_edges) / 2
```

Comment:
@RuneDominik Two things here: 

First, width should be taking normalization into account now, right? 

Second: why is width `diff / 2`? Shouldn't it just be `diff`?

Reply:
As this only applies to cases where only one bin has any entry != 0 we should not need the normalization, as the bin-height in this case is irrelevant. 

`diff/2` was chosen instead of `diff` to mimic a Gaussian `N(bin_mid, diff/2)` and thus taking the bin to represent the +-1sigma interval. Arguably, this results in 32% of the probability is outside the bin, even though the data does not support this. This might be a problem for logarithmic binning. It works fine for linear binning, as the dirac-delta test-cases show.

Reply:
or do `with = np.diff(bin_edges)` and then `std = 0.5 * width`.

One could also just assume a uniform distribution inside the bin, in which case the std would be:
```
np.sqrt(1 / 12 * width**2)
```